### PR TITLE
Model controller test

### PR DESCRIPTION
**Pull request by Emmykage**
the app has a 

-     Splash screen    
-     Sign up and login pages      
-     Home page 
-     Transactions page

https://www.loom.com/share/3845c140ac2e419ba4f650552f8b68d5

### CODE REVIEW

**Comment by Emmykage** [comment]
just fixed linters

File: README.md [hunk 1]
```diff
@@ -1,24 +1,112 @@
...
 
-* Configuration
+- Ruby on Rails
+- Boostrap
+- Postgres SQL
+- Rspec
+- Capybara
 
-* Database creation
+## Live Demo (if available)
```
Comment (codepantha) [inline README.md:19]: - [ ] As part of the project requirements, please deploy the application using Heroku, Netlify, or GitHub Pages and include a link to the deployed project in your README 👍

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @Emmykage 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow correctly
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] You have done a great job so far with the styling of this project, however, it is required that you follow the design guidelines: Please ensure that the colors, fonts, and layout match the given [Behance design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
## Behance Design
image
## Your Design
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Davidosky007) [reply README.md:19]: * These requested change hasn't been fixed kindly fix this as its a[ requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Emmykage 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Nice video presentation 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
* I strongly suggest you Create more unit and integration tests for all the most important components of your RoR application.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (codepantha) [inline README.md:19]: - [ ] You have done well by trying to deploy your application using Heroku. However, the application was not properly deployed as you can see below 👇. Please try to fix the issue with your deployment 🙏 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @Emmykage 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow correctly
✔️ No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Emmykage) [reply README.md:19]: hey.. i have tried everything to fix this but It doesn't work. i really don't know what to do about this

**Review by bimbolabuari — COMMENTED** [review]
Hello @Emmykage  👋, 

_This is an additional review as a result of the second opinion request._ 

-  Nice implementing the changes requested by the previous CRs, however, I strongly suggest you deploy your page as it is a core project requirement, you can try using heroku to deploy your application, it works better with backend deployment. 

Keep rocking 🚀!

File: README.md [hunk 2]
```diff
@@ -1,24 +1,112 @@
...
+## Acknowledgments
+
+- Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella). Published under the [Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/).
+- Linters workflow created by [@anagudelogu](https://github.com/anagudelogu)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (rloterh) [inline README.md:112]: - [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide instructions below it.
Implementing the above change(s) will make your ReadMe stand out even better 😉

File: README.md [hunk 3]
```diff
@@ -1,24 +1,112 @@
...
+bin/rspec
+```
+
+Windows users have to specify ruby in this case:
+
+```
+ruby bin/rspec
+```
+
+## Authors
```
Comment (rloterh) [inline README.md:85]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

**Review by rloterh — COMMENTED** [review]
## Status: Approved :heavy_check_mark:

Hi Morris,  

Your project is complete, good job! There is nothing else to say other than... it's time to merge it 🍾  Congratulations!  🎉💯🌟 

## Highlights 👍
- Correctly deployed application
- Added required models, controllers, and corresponding views
- App is working as intended
- Good test coverage

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**